functions/Test-DbaLastBackup.ps1

```diff
@@ -166,6 +169,7 @@ function Test-DbaLastBackup {
         [switch]$CopyFile,
         [string]$CopyPath,
         [int]$MaxSize,
+        [string[]]$DeviceType,
```

Comment:
Excellent - matches with Get-DbaBackupHistory's type. Looks great, I updated the Tests file to recognize the new param 👍

Reply:
Oh no, I updated the wrong test 😅  One moment, will fix.